connector/spanmetricsconnector/config.go

```diff
@@ -80,6 +83,13 @@ type ExplicitHistogramConfig struct {
 	Buckets []time.Duration `mapstructure:"buckets"`
 }
 
+type Event struct {
+	// Enabled is a flag to enable events.
+	Enabled bool `mapstructure:"enabled"`
+	// Dimensions defines the list of dimensions to be added to events metric
+	Dimensions []Dimension `mapstructure:"dimensions"`
```

Comment:
I think `Dimensions` should not be empty if `Enabled` is `true`. The reason is because if it's empty, it would contain exactly the same data as `calls`, but please correct me if I'm wrong.

If you agree with the above statement, can you add validation (similar to the existing `validateDimensions`) and update tests, please?

Reply:
Yes, you are right.. We need to validate the dimensions. Will add it. Thank you